### PR TITLE
Setup stub amo app

### LINKED ISSUE #701: Create app for mobile pages
This is to start on the mobile pages for amo.

Thinking `amo` makes sense as a name since we can devhub separately for the developer pages.

Relates to mozilla/addons#151

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #701 

Adds a basic hello world app stub and refactors some of the server tests.

### CODE REVIEW

File: tests/server/TestConfig.js [hunk 1]
```diff
@@ -1,53 +1,59 @@
...
   afterEach(() => {
     process.env.NODE_ENV = 'production';
+    delete process.env.NODE_APP_INSTANCE;
   });
 
-  it('should not ever have disableSSR set to true', () => {
-    const config = requireUncached('config');
-    assert.equal(config.get('disableSSR'), false);
-  });
+  for (const appName of appsList) {
```
Comment (tofumatt) [inline tests/server/TestConfig.js]: Nice.

File: tests/server/TestFrameGuardConfig.js [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
     delete process.env.NODE_APP_INSTANCE;
   });
 
-  it('should default frameGuard to "deny"', () => {
-    const config = requireUncached('config');
-    const frameGuardConfig = config.get('frameGuard');
-    assert.equal(frameGuardConfig.action, 'deny');
-    assert.equal(frameGuardConfig.domain, undefined);
-  });
+  for (const appName of appsList) {
```
Comment (tofumatt) [inline tests/server/TestFrameGuardConfig.js]: What's the reason for `const` over `let` in these loops? I suppose the app names should be immutable. Just threw me off. I suppose I answered my own question 😁

**Comment by tofumatt** [comment]
r+

File: tests/server/TestFrameGuardConfig.js (hunk 2 above)
Comment (muffinresearch) [inline tests/server/TestFrameGuardConfig.js]: Using `const` in a for in/of loop head like this creates a new binding for each loop iteration (like let) but the difference is the binding is immutable. In other words the `const` in the for in loop prevents re-assigning `appName` in the body.

This is also enforced by pref-const: http://eslint.org/docs/rules/prefer-const

**Pull request merged**